estilo.yml

```diff
@@ -8,3 +8,8 @@ colorschemes:
   - name: night-owl
     background: 'dark'
     palette: night-owl.vim
+
+lightline:
+  - name: nightowl
```

Comment:
Due to the way the lightline theme is created, I couldn't name this `night-owl` because dashes aren't valid variable names.